test/unit/helpers.js

```diff
@@ -225,3 +227,15 @@ export function imageToString (im) {
 export function makeRepeated (arr, repeats) {
   return [].concat(...Array.from({ length: repeats }, () => arr));
 }
+
+// allows temporary disabling of cache, which can be enabled again
+// with  enableCache()
+export function disableCache () {
+  globalThis.cachePreviousMax = vips.Cache.max();
+  vips.Cache.max(0);
+}
+
+// enables the cache again after it has been disabled with disableCache()
+export function enableCache () {
+  vips.Cache.max(globalThis.cachePreviousMax);
+}
```

Comment:
I think this logic should probally be moved to `test_block.js` instead. For example, you could do this:
```js
  let cachePreviousMax;

  before(function () {
    // Temporary disable the operation cache of libvips to avoid caching blocked operations
    cachePreviousMax = vips.Cache.max();
    vips.Cache.max(0);
  });

  after(function () {
    // Re-enable the operation cache of libvips
    vips.Cache.max(cachePreviousMax);
  });

  afterEach(function () {
    cleanup();
  });
```
(fwiw, the `afterEach`-block is not really necessary, but helps in debugging ref leaks and ensures that the images are properly cleaned up after every test)

Reply:
Makes sense to me. I thought you could only apply those functions globally to all running tests.
Moved in https://github.com/kleisauke/wasm-vips/pull/24/commits/6dcf7925ea43ef89d3837cb3bee55c61dcb3e9e8

Reply:
I initially meant to move it to the `describe` block, but this also seems to work, TIL.

Reply:
(I moved it into the `describe` block with commit ef0088e01814414bc6d6c2e10846952b7f7823af to be consistent with the other tests)